src/main/resources/message/sendUrlMessage.xml

```diff
@@ -0,0 +1,82 @@
+  ~ you may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied. See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+<template xmlns="http://ws.apache.org/ns/synapse" name="sendUrlMessage">
+    <parameter name="receiver" description="Unique viber user id"/>
+    <parameter name="senderName" description="The sender's name to display"/>
+    <parameter name="type" description="The type of the message"/>
+    <parameter name="mediaUrl" description="The url"/>
+    <parameter name="senderAvatarUrl" description="The sender’s avatar URL"/>
+    <parameter name="trackingData" description="Allow the account to track messages and user’s replies"/>
+    <parameter name="minApiVersion" description="Minimal API version required by clients for this message"/>
+    <sequence>
+        <property name="uri.var.receiver" expression="$func:receiver"/>
+        <property name="uri.var.senderName" expression="$func:senderName"/>
+        <property name="uri.var.type" expression="$func:type"/>
+        <property name="uri.var.mediaUrl" expression="$func:mediaUrl"/>
+        <property name="uri.var.senderAvatarUrl" expression="$func:senderAvatarUrl"/>
+        <property name="uri.var.trackingData" expression="$func:trackingData"/>
+        <property name="uri.var.minApiVersion" expression="$func:minApiVersion"/>
+        <payloadFactory media-type="json">
+            <format>
+                {
+                "receiver":"$1",
+                "sender":{
+                "name":"$2"
+                },
+                "type":"$3",
+                "media":"$4"
+                }
+            </format>
+            <args>
+                <arg expression="$.receiver" evaluator="json"/>
+                <arg expression="$.senderName" evaluator="json"/>
+                <arg expression="$.type" evaluator="json"/>
+                <arg expression="$.mediaUrl" evaluator="json"/>
+            </args>
+        </payloadFactory>
+        <script language="js">
+            <![CDATA[
+            payload = mc.getPayloadJSON();
+            var senderAvatarUrl = mc.getProperty("uri.var.senderAvatarUrl");
+            var trackingData = mc.getProperty("uri.var.trackingData");
+            var minApiVersion = mc.getProperty("uri.var.minApiVersion");
+
+            if ( senderAvatarUrl!= null && senderAvatarUrl != "") {
+            payload.sender.avatar = senderAvatarUrl;
+            }
+
+            if( trackingData != null && trackingData != ""){
+            payload.tracking_data = trackingData;
+            }
+
+            if( minApiVersion != null && minApiVersion != ""){
+            payload.min_api_version = minApiVersion;
+            }
+
```

Comment:
Fix all Formatting issues inside the script mediator
Can you check other file and fix